git_buildpackage

```diff
@@ -163,9 +172,27 @@ def create_source_package(repo_dir, output_dir, revision, build_args):
     safe_run(command, cwd=repo_dir, interactive=True)
 
 
+    # Lets find the .dsc
+    #if not os.path.exists(
+    found_dsc = False
+    for fname in os.listdir(absoutput_dir):
+        if fnmatch.fnmatch(fname, '*.dsc'):
+            input_file = os.path.join(absoutput_dir, fname)
+            found_dsc = True
+            with open(input_file, 'a') as dsc_file:
+                dsc_file.write('Git-Id: ' + git_id)
+                dsc_file.write('Git-URL: ' + url)
```

Comment:
I'm fine with having them all enabled by default. But the Git-URL just got created from the post-export hook.